xmake/modules/package/manager/vcpkg/find_package.lua

```diff
@@ -52,6 +52,9 @@ function main(name, opt)
             arch = "x64"
         end
     end
+    if plat == "linux" and arch == "x86_64" then
+        arch = "x64"
+    end
 
```

Comment:
I think we can simplify them if all platform architectures are not x86_64

```lua
    if plat == "macosx" then
        plat = "osx"
    end
    if arch == "x86_64" then
        arch = "x64"
    end
```

Reply:
agree.